tests/as_python_test.py

```diff
@@ -10,6 +10,10 @@ class dummy_class:
     def __init__(self):
         self.i = 0
 
+def gen_tree(depth):
+    if depth == 0: return SExp.to(1337)
```

Comment:
The `flake8` linter doesn't like this, can you put the `return` on the next line?

Reply:
fixed (along with some other flake8 complaints). Do you think we should run flake8 on CI? (and if so, what do you think our max line length should be)